src/main/java/graphql/execution/ValuesResolver.java

```diff
@@ -106,10 +109,15 @@ private Object getVariableValue(GraphqlFieldVisibility fieldVisibility, Variable
     }
 
     public Map<String, Object> getArgumentValues(List<GraphQLArgument> argumentTypes, List<Argument> arguments, Map<String, Object> variables) {
-        return getArgumentValues(DEFAULT_FIELD_VISIBILITY, argumentTypes, arguments, variables);
+        GraphQLCodeRegistry codeRegistry = GraphQLCodeRegistry.newCodeRegistry().fieldVisibility(DEFAULT_FIELD_VISIBILITY).build();
+        return getArgumentValuesImpl(codeRegistry, argumentTypes, arguments, variables);
+    }
```

Comment:
This version is used ONLY during IF directive handling where the schema is not available.